airflow/hatch.toml

```diff
@@ -0,0 +1,11 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["27", "38"]
+version = ["v1", "v2"]
+
+[envs.default.overrides]
+matrix.version.env-vars = "AIRFLOW_VERSION"
+
+[envs.latest.env-vars]
+AIRFLOW_VERSION = "latest"
```

Comment:
@ofek I'm not sure this is the correct way to configure the latest env. Could you take a look?

Reply:
Yup! https://github.com/DataDog/integrations-core/blob/68fe432b9c581d01048ea42009332d0b1560391a/datadog_checks_dev/datadog_checks/dev/tooling/testing.py#L277-L282

Reply:
Thank you!